configs/recommended.js

```diff
@@ -11,6 +11,7 @@ module.exports = Object.assign({}, all, {
     'react/jsx-no-duplicate-props': 2,
     'react/jsx-no-target-blank': 2,
     'react/jsx-no-undef': 2,
+    'react/jsx-props-no-spread-multi': 2,
```

Comment:
Probably bold to suggest this but I have a hard time imaging use-cases for spreading the same expression twice.

Reply:
Adding anything to the recommended config is a breaking change, so we can’t do that.